kedro/framework/session/session.py

```diff
@@ -75,6 +76,12 @@ def _jsonify_cli_context(ctx: click.core.Context) -> Dict[str, Any]:
     }
 
 
+def _clear_hook_manager(hook_manager):
```

Comment:
Minor: does this function belong here or in `kedro/framework/hooks/manager.py` or just inline in `KedroSession.close`? Does it get used anywhere outside that method?

Reply:
In fact, do we need this function at all? See general review comments...